code/datums/components/honkspam.dm

```diff
@@ -0,0 +1,22 @@
+// This used to be in paper.dm, it was some snowflake code that was
+// used ONLY on april's fool.  I moved it to a component so it could be
+// used in other places
+
+/datum/component/honkspam
```

Comment:
Doesn't squeak component do exactly same thing ? What's the difference here ?

Reply:
OH, I will look at the squeak then.  I didn't know there was a noise component

Reply:
Sorry for the delay.  Basically there was code in paper that during april fools, when you pulled a sheet from a paper bin, it would say HONK over and over again and just keep honking at you every time you tried to write to it.  It was a very simple method so I just converted it to a component.  It could be more generalized latter if need be but its designed to be annoying.